rmgpy/molecule/atomtype.py

```diff
@@ -665,7 +665,7 @@ def getFeatures(self):
 atomTypes['F1s'].setActions(incrementBond=[],               decrementBond=[],               formBond=['F1s'],      breakBond=['F1s'],      incrementRadical=['F1s'], decrementRadical=['F1s'], incrementLonePair=[],      decrementLonePair=[])
 
 #these are ordered on priority of picking if we encounter a more general atomType for make
-allElements=['H', 'C', 'O', 'N', 'S', 'Si', 'Cl', 'Ne', 'Ar', 'He', 'X']
+allElements=['H', 'C', 'O', 'N', 'S', 'Si', 'Cl', 'F', 'Ne', 'Ar', 'He', 'X']
```

Comment:
While at it, and although beyond the scope of the PR title, perhaps add `I` as well?
BTW, if Ar is here, should we also consider He and Ne?

Reply:
Good point with `I`. Added in subsequent commit. 
He and Ne are already listed.